src/client/components/date-input/date-input.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../imports';
+
+.date-input {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: 0;
```

Comment:
this violates the 'components should not position themselves' rule, please move the flex stuff into the parent.